sdk/dapp-kit/test/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+	"extends": "../tsconfig.esm.json",
+	"include": ["./**/*", "../src"],
+	"compilerOptions": {
+		"rootDir": "..",
+		"noEmit": true,
+		"moduleResolution": "Node",
```

Comment:
why `Node` vs `nodenext`? if the former, should it be `node10` or does that not matter? just looking at https://www.typescriptlang.org/tsconfig#moduleResolution

Reply:
I can update this to nodenext, this was just copied from one of the other packages